### PR TITLE
Reorganize page structure

### PR DESCRIPTION
**Pull request by yungyuc**
Reorganize all contents into 5 pages and list them in the left side bar.  Avoid putting organized contents in _posts/ directory.  The five pages are ordered by significance and they are:
1. index.md
2. publications/publication.md
3. codes/codes.md
4. links/links.md
5. updates/updates.md

The "posts" in the _posts/ directory look more like information update, so updates/updates.md is created to house them, and I no longer list those posts in the side bar.  I deleted all posts except one placeholder.

Is this a sensible change?  It seems jekyll was designed like a blogging system.  But on the CESE web site I don't expect to do a lot of blogging.  We can still have chronological blog items but it seems more natural to me to organize important information into fixed pages.

### CODE REVIEW

**Comment by tai271828** [comment]
Thanks for the PR. Review is in progress.

jekyll **IS** designed as a blogging system.

**Comment by tai271828** [comment]
- Is "site updates" similar to the idea to host drafts? Or it indeed means some critical updates, e.g. "a new publication is published!"
- Please remove the tag "show" in the post. It is a customized I defined to iterate the order of the side bar. As we want to fix the entries and order, we do not need to use the tag.

This layout and order looks good to me. I will merge this PR after the "show" tag is removed.

**Comment by yungyuc** [comment]
Site update would be like editorial notes.  A notification of a new publication is a good example of its usage.  Therefore the site update won't work like drafting.

Drafting should be shown clearly that it is not part of the website.  Ideally it should be under a different domain, and unreachable be crawlers.

File: _posts/2016-05-29-new-web-site.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+layout: post-no-cover
+title: Prepare a new website of the CESE method
+date: 2016-05-29 07:20:00 +0800
+categories: posts
+show: true
```
Comment (tai271828) [inline _posts/2016-05-29-new-web-site.md:6]: As default.html fixes the side bar order as your above change, the "show" tag is not used anymore.
Please feel free to remove this customized tag.

File: codes/codes.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: post-no-cover
-title: codes
-date:   2013-04-01 00:00:00
-categories: posts
+title: Codes
+date: 2016-05-29 07:03:00 +0800
+permalink: /codes
 show: true
```
Comment (tai271828) [inline codes/codes.md]: Please remove the "show" tag for the same reason above.

File: index.md [hunk 3]
```diff
@@ -2,8 +2,8 @@
 layout: post
 title: Home
 cover: cover.gif
-date:   2014-01-01 00:00:00
-categories: posts
+date: 2016-05-29 07:00:00 +0800
+permalink: /
 show: true
```
Comment (tai271828) [inline index.md]: Please remove the "show" tag for the same reason above.

File: links/links.md [hunk 4]
```diff
@@ -1,11 +1,26 @@
 ---
 layout: post-no-cover
-title: Other Researchers
-date:   2013-07-01 00:00:00
-categories: posts
+title: Links
+date: 2016-05-29 07:05:00 +0800
+permalink: /links
 show: true
```
Comment (tai271828) [inline links/links.md]: Please remove the "show" tag for the same reason above.

File: publications/publications.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: post-no-cover
-title: Publication
-date:   2013-05-01 00:00:00
-categories: posts
+title: Publications
+date: 2016-05-29 07:02:00 +0800
+permalink: /publications
 show: true
```
Comment (tai271828) [inline publications/publications.md]: Please remove the "show" tag for the same reason above.

File: updates/updates.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post-no-cover
+title: Site Updates
+date: 2016-05-29 07:01:00 +0800
+permalink: /updates
+show: true
```
Comment (tai271828) [inline updates/updates.md]: Please remove the "show" tag for the same reason above.

**Comment by tai271828** [comment]
I have commented inline to indicate what the "show" tag means for my previous unclear statement.

**Pull request merged**